scss/base/_variables.scss

```diff
@@ -133,6 +133,7 @@ $SUPER_Z_INDEX:                           20000 !default; //in core, here for bi
 $zindex-fullscreen:                  2147483647 !default; //in screenfull, here for bird's eye view tracking
 $zindex-state-screen:                      1000 !default;
 $zindex-vr-controls:                       1001 !default;
+$zindex-watermark:                        12601 !default;
```

Comment:
How was this value chosen? Would 1001, be enough since the watermark is currently being blocked by `.oo-state-screen-selectable` (which has a z-index of 1000)?

Reply:
\+